src/_help/publish-documentation/deploy-and-release-management.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Deploy and release management
+---
+
+- TOC
+{:toc}
+
+## What are deployments
+
+Deployment corresponds to the publication of an uploaded version of documentation.
+
+When we talk about deployment, we mean the act of publishing a version of your documentation to your API users. A deployed version is a published, live version.
+
+![](/images/help/deployments-list.png)
+
+By default, a version is automatically deployed after being uploaded. However, you can choose to deploy a version manually at a later time by using the Manual Release mode.
+
+You can also add a title and description to your last automatic deployment afterward. This option is available from the page of the deployment, accessible in the list of deployments for your documentation.
+
+### Deploy from the dashboard
+
+To deploy a new version of documentation from the dashboard, you first need to go to the settings of that documentation.
+
+From there, you will find the "Deploy new version" button, which will allow you to choose the file to upload and, if the documentation has branches, the branch to deploy it on.
+
+![](/images/help/upload-version-quick.png)
+
+After processing, the new version will be automatically deployed after a few moments unless you have enabled Manual Release mode.
+
+### Deploy from the CLI
```

Comment:
I'm in favor of replacing CLI/CI/GitHub action paragraphs with links pointing to the right section.

Reply:
Last Monday, we decided during the GTM weekly to share this topic to the team as we weren't sure of what should be the most convenient.

To keep it short, [they suggested ](https://itducks.slack.com/archives/C01DP38BRTL/p1699279553854939) to keep a short explanation with a link pointing to te full section, instead of directly pointing to the complete page elsewhere.

As nobody else jumped in the thread, and because we were looking for such feedback to make a decision, I chose to stick to that so far, awaiting for more reviews on the thread and the PR. This is a reason why this part has not moved yet.